internal/serialization/object_data.go

```diff
@@ -134,6 +134,12 @@ func (o *ObjectDataOutput) WriteInt32(v int32) {
 	o.position += Int32SizeInBytes
 }
 
+func (o *ObjectDataOutput) WriteInt32_(v int32, bo binary.ByteOrder) {
```

Comment:
How about renaming this method to `WriteInt32BigEndian` and removing the `bo` parameter?

Reply:
Makes more sense since we already have LE one, done at https://github.com/hazelcast/hazelcast-go-client/pull/912/commits/b9c72d900a0da806a1b7b419c59514656f2c5b84.